config/locales/en/nsm/steps.yml

```diff
@@ -322,7 +322,9 @@ en:
             process_1: "We'll assess your claim."
             process_2_html: "<p class='govuk-body'>We’ll send you an email to let you know the outcome of your claim or to ask for more information.</p>
```

Comment:
It's making me uncomfortable just how much markup is in our translation files now. Strictly speaking, all the html tags should be in the ERB files, and just the text should be in here. Maybe it's doesn't need to be fixed right away, but if not I think we're approaching the point where we might need a ticket to rework this chunk as a piece of tech debt.

Reply:
May be we should talk about this, I have just followed the current practice, i.e. `config/locales/en/nsm/steps.yml:293`. Personally I completely agree with you.